Source/WTF/wtf/text/Base64.h

```diff
@@ -86,10 +82,10 @@ Vector<uint8_t> base64URLEncodeToVector(std::span<const uint8_t>);
 String base64URLEncodeToString(std::span<const std::byte>);
 String base64URLEncodeToString(std::span<const uint8_t>);
 
-std::optional<Vector<uint8_t>> base64URLDecode(StringView);
-inline std::optional<Vector<uint8_t>> base64URLDecode(ASCIILiteral literal) { return base64URLDecode(StringView { literal }); }
-std::optional<Vector<uint8_t>> base64URLDecode(std::span<const std::byte>);
-std::optional<Vector<uint8_t>> base64URLDecode(std::span<const uint8_t>);
+std::optional<Vector<uint8_t>> base64URLDecode(StringView, OptionSet<Base64DecodeOption> = { });
+std::optional<Vector<uint8_t>> base64URLDecode(ASCIILiteral literal, OptionSet<Base64DecodeOption> = { });
+std::optional<Vector<uint8_t>> base64URLDecode(std::span<const std::byte>, OptionSet<Base64DecodeOption> = { });
+std::optional<Vector<uint8_t>> base64URLDecode(std::span<const uint8_t>, OptionSet<Base64DecodeOption> = { });
```

Comment:
I could see leaving the options argument out of these. If you want to pass unusual options, you could just call the base one and pass the URL flag.